### ISSUE TITLE
[Enhancement] Add HSQLDB support

### ISSUE THREAD

**Issue by Campano** [beekeeper-studio/beekeeper-studio#66]
Not much to say, it would be neat to add [HSQL](http://hsqldb.org/) support for Java developers.

**Comment by rathboma** [comment]
Yeah would love to add this and [H2](https://www.h2database.com/html/main.html) also.

Need to figure out a good way to do this that doesn't suck (requires spinning up a JVM instance). These require the use of the Java driver 

It's not on my immediate to-do, but I'll definitely figure this out. Also need to do it for a Sybase driver.
